saleor/graphql/order/mutations/order_confirm.py

```diff
@@ -60,6 +62,34 @@ def get_instance(cls, info: ResolveInfo, **data):
             )
         return instance
 
+    @staticmethod
+    def charge_transaction_items(user, app, order, manager):
+        for transaction_item in order.payment_transactions.filter(
+            available_actions__contains=[TransactionAction.CHARGE],
+            authorized_value__gt=0,
+        ):
+            charge_value = transaction_item.authorized_value
+            if transaction_item.authorized_value > order.total.gross.amount:
+                charge_value = order.total.gross.amount
+            event = transaction_item.events.create(
+                amount_value=charge_value,
+                currency=transaction_item.currency,
+                type=TransactionEventType.CHARGE_REQUEST,
+                user=user,
+                app=app,
+                app_identifier=app.identifier if app else None,
+                idempotency_key=str(uuid.uuid4()),
+            )
+            request_charge_action(
```

Comment:
It can raise an exception. You should handle it. Also from the business perspective, we should clarify, what should we do in case of having more than a single transaction_item, where one of them raises an exception - do we want to stop the charging process? Stop orderConfirm? or maybe skip the transaction and proceed the rest of them?
I would say, that we should continue the process, and always mark the order as confirmed, as this action was explicitly requested by staff user, but I also think that it is worth to discuss internally with your team.